db/migrate/20200601003833_add_fulfillment_to_item_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddFulfillmentToItemOrders < ActiveRecord::Migration[5.1]
+  def change
+    add_column :item_orders, :fulfilled?, :boolean, default: false 
```

Comment:
default false (i.e. unfulfilled) and then a merchant has the option to change to fulfilled and deplete their inventory.